src/theme/Footer/index.js

```diff
@@ -0,0 +1,89 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+import React from 'react';
+import styles from './styles.module.css'; // CSS modules 
+
+// This file is generated automatically using the `npm run swizzle @docusaurus/theme-classic Footer` command 
+// This is done to modify the default footer 
+// For more information look https://docusaurus.io/docs/migration/manual#footer
+
+const content = {
+  sponsorsContent: (
+    <>
+      If you like our work, you can <b>sponsor & support</b> the community!<br></br>
+      All the money will be used to continue our mission of <b>providing simplified cloud native education to all</b> and make this community reach more number of people.
+    </>
+  )
+}
+
+const logo = {
+  hashnode: { 
```

Comment:
@AvineshTripathi what are your thoughts on using something like react-icons?

Reply:
Any particular reason going for react-icons?